Library/Homebrew/install.rb

```diff
@@ -24,6 +24,7 @@ def attempt_directory_creation
       Keg::MUST_EXIST_DIRECTORIES.each do |dir|
         begin
           FileUtils.mkdir_p(dir) unless dir.exist?
+          FileUtils.touch("#{dir}/.keepme")
```

Comment:
Thinking about it: can also probably do a `unless (dir/".keepme").exist?` here too to avoid touching files that already exist. Sorry I missed that the first time!

Reply:
I thought about it, and since it will just update modification time kept it as is. However, since you have the same thoughts — I have added that check to not overwrite it every time